src/CosmosEditorManager.ts

```diff
@@ -100,6 +106,8 @@ export class CosmosEditorManager {
                         editor = new DocDBDocumentNodeEditor(<IAzureNode<DocDBDocumentTreeItem>>editorNode);
                     } else if (editorNode.treeItem instanceof MongoDocumentTreeItem) {
                         editor = new MongoDocumentNodeEditor(<IAzureNode<MongoDocumentTreeItem>>editorNode);
+                    } else if (editorNode.treeItem instanceof DocDBStoredProcedureTreeItem) {
```

Comment:
Wow this function has major code stink.  But leave it for now.

Reply:
Agreed. I remember asking how we can make this more manageable. Felt weird adding to this.